banditml_pkg/banditml/serving/predictor.py

```diff
@@ -35,6 +35,11 @@ def __init__(
         # the ordered decisions that we need to score over.
         self.decisions = []
 
+        # if using dropout to get prediction uncertainty, how many times to score
+        # the same observation
+        self.num_times_to_score = 50
```

Comment:
is 50 a general best practice / best guess? future hyper param?

Reply:
yea, just a best guess, will need to be less for bigger models

Reply:
the higher the better, but it will add latency to inference